k2/python/tests/linear_fst_with_self_loops_test.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
```

Comment:
Please also add a line to `CMakeLists.txt` below
https://github.com/k2-fsa/k2/blob/1c396b9aee094cd56867dc5c5bd2fb40074ed126/k2/python/tests/CMakeLists.txt#L55